css/css-conditional/idlharness.html

```diff
@@ -16,16 +16,13 @@
 
   <script>
     'use strict';
-    promise_test(async () => {
-      const idl = await fetch('/interfaces/css-conditional.idl').then(r => r.text());
-      const cssom = await fetch('/interfaces/cssom.idl').then(r => r.text());
-      const dom = await fetch('/interfaces/dom.idl').then(r => r.text());
-      const idl_array = new IdlArray();
-      idl_array.add_idls(idl);
-      idl_array.add_dependency_idls(cssom);
-      idl_array.add_dependency_idls(dom);
-      idl_array.test();
-    }, 'Test css-conditional IDL implementation');
+    idl_test(
+      ['css-conditional'],
+      ['cssom', 'dom'],
+      idl_array => {
+        // No objects
+      }
```

Comment:
Nit: consider dropping the callback argument entirely.

Reply:
They were sort of a placeholder for objects to be added, so I've pushed a commit to do just that.